README.md

```diff
@@ -1,8 +1,11 @@
 # pod-browser
 
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
+
 An application for browsing the data on your Solid pod, and for setting ACLs on
 that data.
 
```

Comment:
I found another suggestion, from https://github.com/electron/electron's README. Add:

This project adheres to the Contributor Covenant [code of conduct](https://github.com/inrupt/pod-browser/blob/main/CODE-OF-CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [engineering@inrupt.com](mailto:engineering@inrupt.com).